### ISSUE TITLE
GvrViewer Recenter problem

### ISSUE THREAD

**Issue by Parmenione** [googlevr/gvr-unity-sdk#357]
I have noticed with last release (v 1.0.1) that when we call GvrViewer.Instance.Recenter() the funcion reset the user head but 
without moving the device, the virtual world continue to rotate with a small angle.
With the previous release this problem didn't affect.
To test it using DemoScene and click recenter button from bottom panel.

I hope that you can get here as soon as possible.

GVR-unity-sdk version:1.0.1;
Unity version: 5.4.1;
Devices: Samsung S4 (android 5.0.1) and Samsung S6 (android 6);

**Comment by Tandy-T1000** [comment]
I can verify this problem with release 1.0.1. When app starts there is no drift. When Recenter() called, a  constant rotation is introduced. Only remedy is to restart application. 

GVR Unity : 1.0.1
Unity : 5.4.2.f2
Device: Nexus 6

Please fix soon.

**Comment by Jotamaza** [comment]
I can verify it too. The effect is a huge drifting to the left that appears when GvrViewer.Instance.Recenter() is called, very straightforward. Anyone has a workaround for that?

It looks like it's only happening on Android devices, I haven't seen it on an iPhone 6.

> Please fix soon.

Please, please.

GVR Unity: 1.0.1
Unity: 5.4.2f2
Devices: Samsung Galaxy S6 / Nexus 5

**Comment by jkremin** [comment]
@Parmenione @Tandy-T1000 @Jotamaza 
I've experienced the same drifting problem when the scene is reloaded. 
Delete GvrViewer.Instance.Recenter() from your class then go to the GvrViewer.cs class and comment out the lines 480-484.

```
/*
  public void Recenter() {
    device.Recenter();
  }
*/
```

The drifting on reload is now gone.
It's odd, but it worked for me.

___________
GVR: 1.0.1
Unity: 5.4.2f2
Devices tested: Samsung Galaxy S7 & S6 / Nexus 5

**Comment by Jotamaza** [comment]
I've been testing with GVR 1.10.0 and I still have the same problem. Anyone else?

**Comment by Fenchelbach** [comment]
As I experienced, the "camera drift" happens, when I for example use the "openwebsitefunction", means the app is paused and resumed again and with that the recenterfunction is called.

I am using Unity 5.4.0f1 and the Bug is occuring on Samsung Galaxy S7.
I havent tested it on Iphone yet, but I will.

commenting out the lines as jkremin proposed didn't work for me.

I manually called the Recenter function via button and the bug didn't occure.

I will continue try solving this problem.

**Comment by baroquedub** [comment]
Same bad drift caused by calling the Recentre() method, on Unity 5.5.0f3, GoogleVr 1.0.3, Samsung S6

**Comment by Fenchelbach** [comment]
I just noticed, that the Bug does not happen if I simple "minimize" my App on my Phone and "bring it back to the front" but when I call this function:

 Application.OpenURL();

**Comment by Fenchelbach** [comment]
This also does not happen when using the https://github.com/patcat/GoogleCardboardUnitySDKDemo
But using this and getting all the features of the gvrSDK to work seams a huge effort to me.

**Comment by Fenchelbach** [comment]
Ok it seams that @jkremin  was right, but for me only commenting out `/*
  public void Recenter() {
    device.Recenter();
  }
*/`
Didnt work but I also had to comment out/delete all references to this in the files in:
GoogleVR/Legacy/Scripts/Internal/VRDevices

After that, the Bug did not occur anymore. Thank you @jkremin

**Comment by pencil001** [comment]
I also meet this problem when reload the scene which has used the GvrView. And @jkremin 's workaround did not work for me. 

GVR Unity: 1.0.3 & 1.30
Unity: 5.5.2f1
Devices: Samsung Galaxy Note 5

**Comment by baroquedub** [comment]
Sorry but this is still a problem for me on Unity 5.5.3 / GVR10.3.

Can you confirm whether you're closing because it's fixed in Unity 5.6's built-in GoogleVR support?

**Comment by miraleung** [comment]
This is a hardware specific issue, and Unity C# logic cannot change the rotation. Could you please try filing under the gvr-android-sdk repo?
